src/components/DetailsOverlay/index.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from "@emotion/styled";
+import Filter from "components/Filter";
+import { QueryParams } from "constants/queryParams";
+import { size } from "constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import ButtonRow from "./ButtonRow";
+import SearchRangeInput from "./SearchRangeInput";
+import {
+  CaseSensitiveToggle,
+  ExpandableRowsToggle,
+  FilterLogicToggle,
+  LogV2Toggle,
+  PrettyPrintToggle,
+  WrapToggle,
+} from "./Toggle";
+
+interface DetailsOverlayProps {
+  ["data-cy"]?: string;
+}
+
+const DetailsOverlay: React.FC<DetailsOverlayProps> = ({
+  "data-cy": dataCy,
+}) => {
+  const [filters, setFilters] = useQueryParam<string[]>(
+    QueryParams.Filters,
+    []
+  );
+
+  const deleteFilter = (filterName: string) => {
+    const newFilters = filters.filter((f) => f !== filterName);
+    setFilters(newFilters);
+  };
```

Comment:
Deleting doesn't seem to work


https://user-images.githubusercontent.com/4605522/190255398-94ed05cc-646a-472d-a721-7116597dd2e9.mov

Reply:
Just wanted to surface this again @minnakt  i think this is still a problem